src/app/utils/ledger.ts

```diff
@@ -0,0 +1,26 @@
+import { Account, getMasterFingerPrint } from '@secretkeylabs/xverse-core';
+import { Transport } from '@secretkeylabs/xverse-core/ledger/types';
+
+// eslint-disable-next-line import/prefer-default-export
+export const findLedgerAccountId = async ({
```

Comment:
do we still need this after adding the `deviceAccountIndex` to the accounts interface?

Reply:
Yes, otherwise we should migrate the `ledgerAccountsList` for all users to have this new field filled in with a value.

Reply:
I just added the migration for this field in the PR related to Ledger account removal: https://github.com/secretkeylabs/xverse-web-extension/pull/524/files#diff-b2b9834947cef31784430ce2e167e0b23311b6b635d6f42175aa98b11be66cc4R89